docs/content/guides/3.performance/1.index.md

```diff
@@ -0,0 +1,44 @@
+# Introduction to Web Performance
+::subheader
+Improve your website's user experience and increase conversions by optimizing your website's performance.
+::
+
+Web performance is an ever-important subject in modern web. While your Alokai application is built with the best practices in mind, it's important to understand some of the basics of web performance so that as you build out your custom storefront, you can ensure that your users have a great experience.
+
+On the following pages you will find list of good practices that will help you optimize your website performance.
+
+## Monitoring Performance
+
+### Lighthouse
+
+Lighthouse allows you to test for best SEO, performance, and accessibility practices. These tests run by simulating a user's experience with a specific CPU speed, network speed, and device type. 
+
+These tests run in an isolated environment - so while they can be a way to identify issues, they may not give you a complete picture of how your site performs for real users. 
```

Comment:
suggestion: I would put it as a NuxtContent helper like `tip` / `info` to attract attention as it is quite important information.

Reply:
Also, we could mention here that to audit how the website performs for the real users we could use Chrome User Experience Dashboard (CrUX) to get the Core Web Vitals.

One sentence about it and a link to our docs where the user can read more about it.